polartools/diffraction.py

```diff
@@ -0,0 +1,155 @@
+# Copyright (c) 2020, UChicago Argonne, LLC.
+# See LICENSE file for details.
+
+import numpy as np
+from pandas import DataFrame
+from lmfit.models import GaussianModel, LorentzianModel, VoigtModel, LinearModel, PseudoVoigtModel
+import matplotlib.pyplot as plt
+from .load_data import load_table
+
+def fit_peak(xdata,ydata,model='Gaussian',output=False):
+    """
```

Comment:
The doc string needs to be updated. This goes semi-automatically into the documentation (https://polartools.readthedocs.io/en/latest/)

Reply:
ok